src/protocol/prf_sharding/mod.rs

```diff
@@ -669,6 +670,7 @@ async fn move_single_value_to_bucket<BK, C, S, F>(
     record_id: RecordId,
     bd_key: BitDecomposed<S>,
     value: S,
+    breakdown_count: Option<usize>,
```

Comment:
I don't like this type signature. I'd rather just pass in a usize and have the function check if it's a power of two or not.